plugins/MeteorShowers/src/MeteorShowersMgr.hpp

```diff
@@ -317,10 +322,11 @@ private slots:
 	int m_updateFrequencyHours;
 	QString m_url;
 	QDateTime m_lastUpdate;
-	DownloadStatus m_statusOfLastUpdate;
+	UpdateState updateState;
```

Comment:
This plugin's original developer has followed the habit of naming all _member_ variables with a "m_" prefix. I think you should keep this habit in this plugin for consistency.

Reply:
done.